src/TestFramework/PhpUnit/Coverage/CoverageXmlParser.php

```diff
@@ -74,6 +77,17 @@ public function parse(string $coverageXmlContent): array
         return array_merge(...$coverage);
     }
 
+    private function assertHasCoverage(\DOMXPath $xPath): void
+    {
+        $lineCoverage = $xPath->query('/phpunit/project/directory[@name="/"]/totals/lines');
+
+        Assert::notSame(
+            '0',
+            $lineCoverage->item(0)->getAttribute('executed'),
+            '0 Lines of code were executed during tests. Please check your filters.'
```

Comment:
What are the `filters`? I guess the end user will have the same question. Could you please change the exception message to something more descriptive:

* like filters of Infection or PHPUnit
* possible ways of debugging the issue (enable `--debug`?, looking at the `infection.log`? etc.). Anything that can help